lib/Translation/ExportVerilog/ExportVerilog.cpp

```diff
@@ -1417,6 +1418,56 @@ LogicalResult ModuleEmitter::visitSV(AlwaysOp op) {
   return success();
 }
 
+LogicalResult ModuleEmitter::visitSV(AlwaysFFOp op) {
+  SmallPtrSet<Operation *, 8> ops;
+  ops.insert(op);
+
+  indent() << "always_ff @(" << stringifyEventControl(op.clockEdge()) << " "
+           << emitExpressionToString(op.clock(), ops);
+  if (op.resetStyle() == ResetType::AsyncReset) {
+    os << " or " << stringifyEventControl(*op.resetEdge()) << " "
+       << emitExpressionToString(op.reset(), ops);
+  }
+  os << ')';
+
+  // Build the comment string, leave out the signal expressions (since they
+  // can be large).
+  std::string comment;
+  comment = "always_ff @(";
+  comment += stringifyEventControl(op.clockEdge());
+  if (op.resetStyle() == ResetType::AsyncReset) {
+    comment += " or ";
+    comment += stringifyEventControl(*op.resetEdge());
+  }
+  comment += ')';
```

Comment:
nit: This is mostly duplicated in the above code. Does it make sense to unify these two?